jobtracker.go

```diff
@@ -46,18 +46,14 @@ func hadoopIDs(id string) (string, jobID) {
 }
 
 type jobTracker struct {
-	jobClient       RecentJobClient
-	clusterName     string
-	jobs            map[jobID]*job
-	jobsLock        sync.Mutex
-	rm              string
-	hs              string
-	ps              string
-	namenodeAddress string
-	running         chan *job
-	finished        chan *job
-	backfill        chan *job
-	updates         chan *job
+	jobClient   RecentJobClient
+	clusterName string
+	jobs        map[jobID]*job
+	jobsLock    sync.Mutex
+	running     chan *job
+	finished    chan *job
+	backfill    chan *job
+	updates     chan *job
```

Comment:
nit: it's preferred not to futz with column widths to keep the diffs comprehensible (or to do in a separate commit otherwise)